src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax.figures;
+
+public class Rectangle extends Figure {
```

Comment:
Optional to think about: In my mind Triangle and RightTriangle have a kind of relation, maybe it makes sence to create inheritance between them